js-plugin/src/main/kotlin/org/jetbrains/research/deepbugs/javascript/ide/inspections/specific/math/JSDeepBugsMathCallExprInspection.kt

```diff
@@ -0,0 +1,50 @@
+package org.jetbrains.research.deepbugs.javascript.ide.inspections.specific.math
+
+import com.intellij.lang.javascript.library.JSLibraryUtil
+import com.intellij.lang.javascript.psi.JSCallExpression
+import com.intellij.lang.javascript.psi.JSReferenceExpression
+import com.intellij.psi.NavigatablePsiElement
+import com.intellij.psi.PsiElement
+import org.jetbrains.research.deepbugs.javascript.ide.inspections.DeepBugsInspectionManager
+import org.jetbrains.research.deepbugs.javascript.ide.inspections.base.JSDeepBugsCallExprInspection
+import org.jetbrains.research.deepbugs.javascript.ide.inspections.specific.SpecificInspectionDescriptor
+
+abstract class JSDeepBugsMathCallExprInspection : JSDeepBugsCallExprInspection() {
+    init {
+        DeepBugsInspectionManager.register(
+            object : SpecificInspectionDescriptor {
```

Comment:
Can `SpecificInspectionDescriptor` be class with lambda parameter? Instead of interface